practices/accessibility-checklist.md

```diff
@@ -83,6 +83,11 @@ Be sure to create an issue or submit a pull request if you have any questions, o
    [WCAG 1, guideline 14](http://www.w3.org/TR/WCAG10/wai-pageauth.html#gl-facilitate-comprehension)  
    (Also see [WCAG 1 core technique 5: "Comprehension"](http://www.w3.org/TR/WCAG10-CORE-TECHS/#comprehension))
 
+1. **Where opening a hyperlink in a new window cannot be avoided, has an advanced warning been provided?**  
+   Unrequested new windows may cause confusion or disorientation for users with cognitive or visual disabilites. Opening links in new windows is discouraged by both WCAG 1.0 and WCAG 2.0. Does the link have a visible marker indicating its behaviour? Is the marker also accessible to screenreaders?  
```

Comment:
I think it's ok to remove. The links you provided allow the user to read up on it further if they want to understand the reasoning.